### PR TITLE
Add Spire Phantasm, Agent of Acquisitions, Cogwork Librarian, Leovold's Operative

### PR DESCRIPTION
**Pull request by tehdiplomat**

### CODE REVIEW

File: forge-gui/src/main/java/forge/gamemodes/limited/DraftPack.java [hunk 1]
```diff
@@ -0,0 +1,170 @@
+package forge.gamemodes.limited;
+
+import forge.item.PaperCard;
+
+import java.util.*;
+
+public class DraftPack implements List<PaperCard> {
```
Comment (tool4ever) [inline forge-gui/src/main/java/forge/gamemodes/limited/DraftPack.java:7]: this looks like something to better implement with decorator pattern
see TargetChoices class for clean approach

Comment (tehdiplomat) [reply forge-gui/src/main/java/forge/gamemodes/limited/DraftPack.java:7]: Ah good. Ruby has delegate but I couldn't figure out what it was called in Java

File: forge-gui/src/main/java/forge/gamemodes/limited/LimitedPlayer.java [hunk 2]
```diff
@@ -541,11 +629,51 @@ public boolean handleIllusionaryInformant() {
...
+        playerFlags |= LeovoldsOperativeExtraDraft;
+        return true;
+    }
+
+    public boolean handleCogworkLibrarian(DraftPack pack, PaperCard drafted) {
+        return !Objects.equals(SGuiChoose.one("Draft an extra pick with Cogwork Librarian?", Lists.newArrayList("Yes", "No")), "No");
+    }
+
+    public boolean handleAgentOfAcquisitions(DraftPack pack, PaperCard drafted) {
+        return !Objects.equals(SGuiChoose.one("Draft the rest of the pack with Agent of Acquisitions?", Lists.newArrayList("Yes", "No")), "No");
```
Comment (tehdiplomat) [inline forge-gui/src/main/java/forge/gamemodes/limited/LimitedPlayer.java:654]: This needs to be overridden by LimitedPlayerAI

File: forge-gui/src/main/java/forge/gamemodes/limited/BoosterDraft.java [hunk 3]
```diff
@@ -463,7 +468,17 @@ protected void computerChoose() {
         for (int i = 1; i < N_PLAYERS; i++) {
             LimitedPlayer pl = this.players.get(i);
             // TODO Agent of Acquisitions activation to loop the entire pack?
```
Comment (tehdiplomat) [inline forge-gui/src/main/java/forge/gamemodes/limited/BoosterDraft.java:470]: Remove TODO

**Review by tool4ever — APPROVED** [review]

**Pull request merged**